Src/FluentAssertions/Primitives/IStringMismatchValidator.cs

```diff
@@ -0,0 +1,10 @@
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Primitives;
+
+internal interface IStringMismatchValidator
```

Comment:
🔧 This interface needs some documentation
🤔 What about `IStringComparisonStrategy` to emphase we're using a Strategy Pattern here?

Reply:
> This interface needs some documentation

I meant both the interface _and_ its properties ;-)

Reply:
Sorry, it seems I was too quick :-)